### ISSUE TITLE
Issue with TorchMD-Net model when used in OpenMM-Torch

### ISSUE THREAD

**Issue by eva-not** [torchmd/torchmd-net#265]
Posting the same issue in the openmm-torch repo as it might be a joint issue.

I am trying to train a NNP on chignolin trajectories in explicit solvent with TorchMD-Net, which I then plan to use in OpenMM as a TorchForce in order to run implicit solvent simulations with hopefully better accuracy. The idea is similar to https://www.nature.com/articles/s41467-023-41343-1 but without the coarse graining. To train the NNP, I have extracted the coordinates and forces of all protein atoms from my explicit solvent trajectory as npy arrays, and also the partial charges of the protein atoms to use as embeddings, again as a npy array (files too large to be attached). 

I trained the NNP following the tutorials from https://github.com/torchmd/torchmd-protein-thermodynamics. The training config yaml file is attached. The batch size was set to 16 as batch sizes higher than that caused GPU memory issues. Once training was done, I selected one of the saved pytorch .cpkt checkpoints and converted it to a pytorch .pt model with:

`from torchmdnet.models.model import load_model`
`model = load_model('train_light/epoch=93-val_loss=8.5159-test_loss=2.0015.ckpt')`
`torch.jit.script(model).save('test.pt')`
The .cpkt and .pt files are attached.

I then attempt to use this model to run MD simulations in OpenMM:
`from openmm.app import *`
`from openmm import *`
`from openmm.unit import *`
`from sys import stdout`
`import numpy as np`
`from openmmtorch import TorchForce`

`prmtop = AmberPrmtopFile('chignolin_dry.prmtop')`
`inpcrd = AmberInpcrdFile('chignolin_dry.inpcrd')`
`system = prmtop.createSystem(nonbondedMethod=NoCutoff)`

.prmtop and .inpcrd attached.

`torch_force = TorchForce('test.pt')`
`system.addForce(torch_force)`
`integrator = LangevinMiddleIntegrator(298.15*kelvin, 1/picosecond, 0.002*picoseconds)`
`platform = Platform.getPlatformByName('CUDA')`
`simulation = Simulation(prmtop.topology, system, integrator, platform)`
`simulation.context.setPositions(inpcrd.positions)`
`simulation.minimizeEnergy()`
`simulation.reporters.append(DCDReporter('output.dcd', 1000))`
`simulation.reporters.append(StateDataReporter(stdout, 1000, step=True,
        potentialEnergy=True, temperature=True))`
`simulation.step(100000)`

but:
![image](https://github.com/openmm/openmm-torch/assets/90212724/93a482db-36e1-458e-bbd5-556d8f673509)

The full OpenMM exception line is:
`OpenMMException: forward() is missing value for argument 'pos'. Declaration: forward(__torch__.torchmdnet.models.model.___torch_mangle_0.TorchMD_Net self, Tensor z, Tensor pos, Tensor? batch=None, Tensor? q=None, Tensor? s=None, Dict(str, Tensor)? extra_args=None) -> ((Tensor, Tensor?))`

Any help would be greatly appreciated!
[files.zip](https://github.com/openmm/openmm-torch/files/14067571/files.zip)

**Comment by eva-not** [comment]
Issue in openmm-torch repo: https://github.com/openmm/openmm-torch/issues/130

**Comment by RaulPPelaez** [comment]
This is an OpenMM-Torch issue, lets discuss there.
